syntax = "proto3";

package example;

// 定义一个消息类型
message Person {
  string name = 1;
  int32 id = 2;
  string email = 3;
}

// 定义一个服务
service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

// 请求消息
message HelloRequest {
  string name = 1;
}

// 响应消息
message HelloReply {
  string message = 1;
}
